manager/src/main/resources/service_limits.json

```diff
@@ -0,0 +1,22 @@
+{
+  "service_limit_instances": [{
+    "instance_type": "EVAL",
+    "processor_limit": "6",
+    "bridge_duration": "50505",
```

Comment:
Would it have sense to use ISO 8601 for this duration, to make it human readable and avoid possible typos/miscalculations?
http://www.java2s.com/Tutorials/Java/java.time/Duration/2960__Duration.toString_.htm
https://en.wikipedia.org/wiki/ISO_8601

Reply:
its duration in seconds. We calculate expite time by `now() + bridge_duration`

Reply:
yes, though I was thinking whether it would have sense to provide value which would tell reader what is the duration without a need to start calculator and divide the provided number to know what does it mean in hours/days.

Reply:
done.. I had made the changes to accept ISO 8601. Now bridge duration is define like below :
```
"bridge_duration": "PT12H",
```